14.serial_search/serial_search.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'date'
+
+def serial_search(input)
+  year = 1900 + (input[0] + input[4]).to_i
```

Comment:
```suggestion
  year = "19#{input[0]}#{input[4]}).to_i
```

好みの問題ですが、こっちの方がなんとなくRubyっぽくて、若干コードの意図もわかりやすいかも？